docs/Architecture Decision Record/016-developer-tabletops.md

```diff
@@ -17,7 +17,7 @@ The purpose of these meetings is to get real-time feedback on the issue in quest
 
 While this will require more scheduling and adds process, ultimately, we have seen greater knowledge transfer for each review and found issues that would have gone missed without this face-to-face feedback that discussions create.
 
-## Q&A
+## Notes
```

Comment:
recommend dropping this as an ADR since this is mostly about how we work. instead suggest moving to technical documentation subdir. the following headings seem worth keeping in the new md:
- Purpose 
- What happens during these meetings?
- Who should be invited? 
- How long are these?